src/summary/documentation/date-helper.tsx

```diff
@@ -0,0 +1,6 @@
+var moment = require("moment");
```

Comment:
might be better to use `dayjs` instead as it is more lightweight than `moment`. Also `dayjs` is already used in the patient-chart to calculate age, so it's already included in the dependencies.

Reply:
Thanks. I replaced that with timeago and works well but i could get only a date like : 2 years ago but in figma we have something like 04-Jan 2018. I'm doing something wrong?

Reply:
@Andu033 You can have a check, if date is not today then use normal date formatting, if date is today then use time ago.

Reply:
@fatmali ok. I will try that. Thanks Fatma.